gravity-forms/gw-create-coupon.php

```diff
@@ -185,7 +185,8 @@ public function create_coupon_wc( $coupon_name, $coupon_code, $amount, $type, $e
 
 		$start_date = rgar( $this->_args['meta'], 'start_date' );
 		if ( $start_date === '' || ! strtotime( $start_date ) ) {
-			$start_date = gmdate( 'Y-m-d H:i:s' );
+			$date       = current_datetime();
+			$start_date = $date->format( 'Y-m-d H:i:s' );
```

Comment:
I think this can be simplified to changing `post_date` on L198/199 to `post_date_gmt`

Reply:
@claygriffiths My apologies, I didn't clarify in the PR. This case only applies if the end user doesn't explicitly set a `start_date`. Modifying L198 would make all coupons available immediately and remove scheduling functionality.

Reply:
Ahh, good point! ✅